prac_08/KivyDemos-master/squaring.kv

```diff
@@ -0,0 +1,22 @@
+BoxLayout:
+    orientation: 'vertical'
+
+    BoxLayout:
+#        orientation: 'vertical'
+
+        TextInput:
+            id: input_number
+            text: '7'
+            font_size: 48
+            multiline: False
+        Button:
+            text: 'Square'
+            # the following line specifies the function in the app class to call when the button is pressed
```

Comment:
I think this comment is required as on_press is self explanatory

Reply:
I think it depends - on_press can be confusing when you pass arguments (such as text) into a method. In this case, I didn't explain this particular call sufficiently.